core/src/Serial/Kokkos_Serial.hpp

```diff
@@ -72,6 +72,8 @@ class SerialInternal {
 };
 }  // namespace Impl
 
+struct NewInstance {};
```

Comment:
This should be

```c++
struct NewInstance { explicit NewInstance() = default; };
```

to disallow calling it via:

```c++
Serial({});
```

Reply:
297e7e3